ccpa_cmplibrary/src/main/java/com/sourcepoint/ccpa_cmplibrary/UserConsent.java

```diff
@@ -16,10 +16,15 @@
     }
 
     public ConsentStatus status;
-    public ArrayList<String> rejectedVendors = new ArrayList();
-    public ArrayList<String> rejectedCategories = new ArrayList();
+    public ArrayList<String> rejectedVendors ;
+    public ArrayList<String> rejectedCategories;
     public JSONObject jsonConsents = new JSONObject();
 
+    public UserConsent(){
+        this.rejectedVendors = new ArrayList();
+        this.rejectedCategories = new ArrayList();
```

Comment:
better to initialise the `jsonConsents` field here as well, just as you did on GDPR

Reply:
@leodoin will do this.